phabfive/passphrase.py

```diff
@@ -44,6 +44,17 @@ def get_secret(self, ids):
             # is INFO, even if env PHABFIVE_DEBUG=1
             log.debug(json.dumps(response["data"], indent=2))
 
+        # When Conduit Access is not accepted for Passphrase the "response" will return value "noAPIAccess" in key "material" instead of the secret
+        if (
+            "noAPIAccess"
+            in response["data"].get(next(iter(response["data"])))["material"]
+        ):
```

Comment:
It is not pretty to have multiple lines in a if case. Extract out `next(iter(response['data']))` to a variabel so you can have the if check on one line and then reuse the result of that variabel in the exception as well.

Reply:
Maybe even pull out the entire `response["data"].get(next(iter(response["data"])))["material"]` into a variabel before the if case